### ISSUE TITLE
$server->loadAndCheckAttestationResponse crashes with TypeError (since 3.3)

### ISSUE THREAD

**Issue by arokettu** [web-auth/webauthn-framework#176]
```php
$server = new Server(...);

$publicKeyCredentialSource = $server->loadAndCheckAttestationResponse(
    $request->getParsedBodyParam('credential'),
    $publicKeyCredentialCreationOptions,
    $request
);
```

Works in 3.2, crashes in 3.3 with
```
Argument 1 passed to Webauthn\AttestationStatement\AndroidSafetyNetAttestationStatementSupport::enableApiVerification() must be an instance of Psr\Http\Client\ClientInterface, null given, called in /app/vendor/web-auth/webauthn-lib/src/Server.php on line 336
```

It probably needs a check whether $this->googleApiKey is actually set. I don't use it and I don't need it

**Comment by lukewarlow** [comment]
It isn't just this line that's broken, not setting a logger is also broken and I imagine other stuff might be too. https://github.com/web-auth/webauthn-framework/issues/177

**Comment by Spomky** [comment]
Hi,

This issue should be fixed with the `v3.3.4`.
Can you confirm it is fine for you now?

**Comment by arokettu** [comment]
3.3.3 works for me, 3.3.4 crashes on `PublicKeyCredentialSource::createFromArray(PublicKeyCredentialSource::jsonSerialize())` with $otherUI is unset or something, but that's a separate issue

This issue can be closed

I need to look at my work project tomorrow to open this new one, but I'm currently satisfied with locking to 3.3.3
